packages/pycdfpp/test_pycdfpp.py

```diff
@@ -0,0 +1,11 @@
+from pytest_pyodide import run_in_pyodide
+
+
+@run_in_pyodide(packages=["pycdfpp", "requests"])
+def test_pycdfpp(selenium):
+    import pycdfpp
+    import requests
+    cdf = pycdfpp.load(requests.get('https://cdaweb.gsfc.nasa.gov/pub/software/cdawlib/0MASTERS/a1_k0_mpa_00000000_v01.cdf').content)
```

Comment:
Could you change the unittest so that we don't make requests to an external website everytime? I think you can make a very small test file and load it from the filesystem instead.